bin/mocha

```diff
@@ -19,50 +20,60 @@ getOptions();
 process.argv.slice(2).forEach(arg => {
   const flag = arg.split('=')[0];
 
+  // every flag UNSHIFTED onto the `args` array is
+  // an flag for the `node` executable; every flag
+  // PUSHED onto the `args` array is a Mocha flag.
+
   switch (flag) {
+    // non-standard debug flags.
+    // any debug-related flag needs Mocha's --no-timeouts
+    // or Mocha will timeout while the debugger is paused.
     case '-d':
+    case 'debug':
+      // rename flag
       args.unshift('--debug');
       args.push('--no-timeouts');
       break;
-    case 'debug':
     case '--debug':
     case '--debug-brk':
     case '--inspect':
     case '--inspect-brk':
       args.unshift(arg);
       args.push('--no-timeouts');
       break;
+    // non-standard flag
     case '-gc':
-    case '--expose-gc':
+      // rename flag
       args.unshift('--expose-gc');
       break;
-    case '--gc-global':
-    case '--es_staging':
-    case '--no-deprecation':
-    case '--no-warnings':
-    case '--prof':
-    case '--log-timer-events':
-    case '--throw-deprecation':
-    case '--trace-deprecation':
-    case '--trace-warnings':
-    case '--use_strict':
-    case '--allow-natives-syntax':
-    case '--perf-basic-prof':
-    case '--napi-modules':
-      args.unshift(arg);
-      break;
     default:
-      if (arg.indexOf('--harmony') === 0) {
-        args.unshift(arg);
-      } else if (arg.indexOf('--trace') === 0) {
-        args.unshift(arg);
-      } else if (arg.indexOf('--icu-data-dir') === 0) {
-        args.unshift(arg);
-      } else if (arg.indexOf('--max-old-space-size') === 0) {
-        args.unshift(arg);
-      } else if (arg.indexOf('--preserve-symlinks') === 0) {
+      if (
+        envFlags.indexOf(arg) > -1 ||
+        arg.indexOf('--harmony') === 0 ||
+        arg.indexOf('--preserve-symlinks') === 0 ||
+        arg === '--gc-global' ||
+        arg === '--es_staging' ||
+        arg === '--expose-gc' ||
+        arg === '--log-timer-events' ||
+        arg === '--use_strict' ||
```

Comment:
Does this support `--use-strict`?

Reply:
No; it never did.  `--v8-use-strict` is allowed, however.